Specification-Draft.MD

```diff
@@ -1,266 +1,152 @@
-# FDC3 Context Objects - Version 0.0.1 #
-## Context Object Overview ##
+# FDC3 Context Data #
 
-* To interoperate, Apps need to exchange commonly recognized context structures that can indicate topic with any number of identifiers or mappings to different systems.  
-* Exchanging context is the most basic entry point to desktop interoperability.  The barriers to adoption for this interaction must be kept as low as possible.
-* Best approach is to transmit as many known identifiers as possible and recipients take what they can understand and ignore the rest.
-* Rendering of context is best left to end points, and it should be left to end points to provide the reference data required for that rendering.
-* Metadata provided with context identifiers should have the following goals:
-  * Facilitate routing of the context to Apps that can accept it.
-  * Allow providing for some basic, human readable cues for end-user lead ambiguity resolution.
+## Context Data Overview ##
 
-## Other Standards ##
-FDC3 recognizes that there are other object definitions for providing context between applications.  Most, if not all of these definitions though are platform specific.  FDC3, as a rule, sets out to be platform agnostic and focused on creating bridges between the various walled gardens on the financial desktop.  
+To interoperate, apps need to exchange commonly recognized context structures that can indicate topic with any number of identifiers or mappings to different systems.
 
-## The Context Envelope ##
+Exchanging context is the most basic entry point to desktop interoperability.  The barriers to adoption for this interaction must be kept as low as possible.
 
-```javascript
-{
-	"object": (string), //object type
-	"definition": (string) //uri of object type definition
-	"version": (string) //version # (semver)
-	"data": (array) //the context payload
-}
-```
+There are two main use case for exchanging context data:
 
-### Examples ###
-#### FDC3 Context Object ####
-```javascript
-{
-   "object" : "fdc3-context",
-   "definition" : "https://fdc3.org/context/1.0.0/",
-   "version" : "1.0.0",
-   "data" :
-       [
-          {
-             "type" : "security",
-    	 "name" : "Apple",
-	 "id" : 
-               {  
-                  "ticker" : "aapl"
-                   "ISIN" : "US0378331005",
-                   "CUSIP" : "037833100",
-                   "FIGI" : "BBG000B9XRY4",
-                   "default" : "aapl"
-               }
-          }
-      ]
-}
-```
+* __Transmitting reference data between applications.__
+  The source application will send as many known identifiers as possible, and the target application will try to match the entity based on the identifiers. It may then choose to map to its own internal domain representation for rendering purposes.
 
```

Comment:
Do we think there will always be some mandatory ID fields for certain Context types?  Or will it be okay to construct an entity with only custom identifiers?  For instance, let's say I have an internal context ID for a Security that I know will only work with one other vendor.  Will this be legal:

{ 
  $type: "fdc3.instrument",
  name: "Apple",
  id: {
    customIdentifier: "A1234"
  }
}

And it's expected that other applications will gracefully handle not being able to process any of the passed in identifiers?

Reply:
From the proposal this would be legal. The more identifiers the originator provides, the more likely it will be that the target application can understand and react to the context.

Reply:
It is also worth pointing out that under the proposal _no identifiers_ would be valid as well, e.g. for sending an order creation request, before you have an order id. This PR tries to extend the current schema so that it allows for the use case where you primarily transfer reference data (i.e. identifiers), as well as the the use case where you want to send structured data.